lib/rules/links/reliability.js

```diff
@@ -0,0 +1,21 @@
+const self = {
+    name: 'links.reliability'
+,   section: 'document-body'
+};
+
+exports.name = self.name;
+const unreliableDomains = ['w3c-test.org', 'www.w3c-test.org'];
+
+exports.check = function (sr, done) {
+    sr.jsDocument.querySelectorAll("a").forEach(function (element) {
+        const url = new URL(element.href);
```

Comment:
There's an issue with relative URLs. See https://nodejs.org/api/url.html#url_new_url_input_base

```suggestion
        const url = new URL(element.href, sr.url);
```

Reply:
When checking a local file (`validate?file=`) in test:
https://github.com/w3c/specberus/blob/c643b1afe879f1ef8b9e28f78431a3620d8a1961/test/api.js#L119

sr.url is `null` because it's cleared:
https://github.com/w3c/specberus/blob/c643b1afe879f1ef8b9e28f78431a3620d8a1961/lib/validator.js#L26

which still breaks `new URL()`. So I add a try catch and done() in catch, to not check the domain here.